service/build.gradle

```diff
@@ -35,6 +35,9 @@ dependencies {
     implementation "bio.terra:terra-common-lib" // 0.0.94-SNAPSHOT is defined in bio.terra.pipelines.java-common-conventions.gradle
     implementation "bio.terra:terra-resource-buffer-client:0.4.3-SNAPSHOT"
 
+    // hk2 is required to use cbas client, but not correctly exposed by the client
+    implementation group: "org.glassfish.jersey.inject", name: "jersey-hk2"
```

Comment:
this is pretty weird, i got this from workspace manager https://github.com/DataBiosphere/terra-workspace-manager/blob/a376aad7a021f648d7d688ba9d0555ca54ff9f1a/service/gradle/dependencies.gradle#L42 but I think we should be able to fix it on the cbas client end

Reply:
should we make tickets to make the fix in the cbas client and update this as necessary?

Reply:
yup - https://broadworkbench.atlassian.net/browse/TSPS-147